client/src/containers/home/StatisticFragment.js

```diff
@@ -1,17 +1,35 @@
 import React from 'react';
 import { Row, Col, Card } from 'antd';
-import { Trans } from '@lingui/react';
+import { Trans } from '@lingui/macro';
+
+import { ReactComponent as PeopleRisk } from '../../images/user-solid.svg';
+import { ReactComponent as EvaluatedBuildings } from '../../images/home-solid.svg';
+import { ReactComponent as ConsolidatedBuildings } from '../../images/house-damage-solid.svg';
```

Comment:
It should be the other way around. Consolidated buildings are fixed while evaluated are damaged. 
```
EvaluatedBuildings } from '../../images/house-damage-solid.svg';
ConsolidatedBuildings } from '../../images/home-solid.svg';
```